packages/services/f-feature-management/src/index.js

```diff
@@ -0,0 +1,34 @@
+function logValueRequest (func, key, value) {
+    console.log(`${func.name}: ${key} => ${value}`); // es-lint-disable
+}
+
+function getBooleanValue (key) {
```

Comment:
Given JS is dynamically typed I wonder if we need separate functions per return type.  Would make sense with TypeScript but not sure with JS.  Might be worth having an additional function for any type. (Just called `getValue()` I guess.)

Interestingly Optimizely have introduced a "generic" version: https://docs.developers.optimizely.com/full-stack/docs/get-feature-variable-javascript#section-generic

Reply:
It depends if/how we want to verify that the type is correct for a given key.

For the other SDKs, you'll get a `null`/`nil` value back, which is essentially declaring an error state (as type enforcement requires you to handle it), so the equivalent here would be something like throwing an error ... which we definitely don't want to do IMO.

If we're putting the onus on the consumer to always type check, I'm happy to reduce to one method (although, I'd place a reasonable bet someone will get burned by truthy checking an integer or string at some point).

Reply:
I think JS engineers are just used to that.  They'll use `=== true` most of the time I expect. But OTOH they can embrace truthiness if standards permit.  E.g. empty string or 0 could mean "this is turned off" which is idiomatic for JS.  Maybe just include all of them, then people get the option?  Or we can wait and see what comments this PR gets 😄

Reply:
I think I like there being different methods for each return type, as long as we guarantee that type. It would help with readability too and make sure the developer knows what kind of value they're working with.

If this were typescript, on the other hand, then I'd be happy with something like `getValue<T>`.

Reply:
We've discussed outside of this thread and we're going to add both the generic and non-generic methods.